src/harness/fourslashImpl.ts

```diff
@@ -2255,27 +2260,21 @@ export class TestState {
             + Harness.IO.newLine()
             + Harness.IO.newLine()
             + this.getSemanticDiagnosticBaselineText(files);
-        Harness.Baseline.runBaseline(this.getBaselineFileNameForContainingTestFile(), result);
+        this.runBaseline(this.getBaselineFileNameForContainingTestFile(), result);
     }
 
     private getSyntacticDiagnosticBaselineText(files: Harness.Compiler.TestFile[]) {
         const diagnostics = ts.flatMap(files,
             file => this.languageService.getSyntacticDiagnostics(file.unitName)
         );
-        const result = `Syntactic Diagnostics for file '${this.originalInputFileName}':`

```

Comment:
I removed this since this would mention `this.originalInputFileName` **again** but perhaps I should replace this with something like:
```
// === Syntactic Diagnostics ===
```

Would that convention be right here?